test/tests/bcs/ad_function_dirichlet_bc/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  issues = '#13261'
+  design = 'bcs/ADFunctionDirichletBC.md'
+  [./ad]
+    requirement = 'The system shall support AD Dirichlet type boundary conditions defined by a function'
+    [./test]
+      type = 'Exodiff'
+      input = 'test.i'
+      exodiff = 'test_out.e'
+      detail = ''
```

Comment:
I don't like this. I'm fairly certain this will produce an empty bullet in the documentation system.

How about you take the "by a function" off the end of your requirement and put that in the detail instead?